src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/rst.stream.last.frame/server.rpt

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()} 
+property newServerConnectRef ${newClientAcceptRef} # external scope
+
+property serverTransport "nukleus://target/streams/http2#source"
+
+accept ${serverTransport}
+        option nukleus:route ${newServerConnectRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:header(":method", "POST")}
+                       ${http:header(":scheme", "http")}
+                       ${http:header(":path", "/")}
+                       ${http:header(":authority", "localhost:8080")}
+connected
+
+write nukleus:begin.ext ${http:header(":status", "200")}
+                        ${http:header("server", "CERN/3.0 libwww/2.17")}
+                        ${http:header("date", "Wed, 01 Feb 2017 19:12:46 GMT")}
+                        ${http:header("content-type", "text/html; charset=UTF-8")}
+                        ${http:header("content-length", "113")}
+write flush
+
```

Comment:
Suggest removing `write nukleus:begin.ext` and `write flush` if not needed to enforce test scenario.